Sources/SwiftUIBackports/iOS/Presentation/Detents.swift

```diff
@@ -169,7 +169,7 @@ private extension Backport.Representable {
         override func willMove(toParent parent: UIViewController?) {
             super.willMove(toParent: parent)
             if let controller = parent?.sheetPresentationController {
-                if controller.delegate !== self {
+                if controller.delegate !== self && _delegate != nil {
```

Comment:
Not sure this is a valid fix, if `_delegate` is NOT nil, then set it??
Feels like this would fail the condition always and never get set as required?

Reply:
Good catch! It was intended to be just the other way around: Only assign _delegate if it does not already have a value assigned. I changed it and also verified, that it fixes the issue.